assets/js/blocks/cart-checkout-shared/payment-methods/express-payment-methods.js

```diff
@@ -75,12 +75,12 @@ const ExpressPaymentMethods = () => {
 	 * This restores the active method and returns the state to pristine.
 	 */
 	const onExpressPaymentClose = useCallback( () => {
-		__internalSetPaymentPristine();
+		__internalSetPaymentIdle();
```

Comment:
In the new diagram I kept this pristine because I assumed on abort the status would be reset to pristine, and subsequent requests would start over. Do you think pristine is more appropriate here for payments?

Reply:
I think Idle is more suitable as pristine indicates a first load. We shouldn't be resetting to pristine at all IMO